reana_client/cli/files.py

```diff
@@ -506,30 +506,43 @@ def move_files(ctx, source, target, workflow, access_token):  # noqa: D301
 @check_connection
 @add_access_token_options
 @click.option("-s", "--summarize", is_flag=True, help="Display total.")
+@click.option(
+    "--filter",
+    "filters",
+    multiple=True,
+    help="Filter results to show only file sizes that match certain filtering "
```

Comment:
Minor: I guess it would be more clear to say `Filter results to show only files that match certain filtering", WDYT?